requirements.txt

```diff
@@ -1,7 +1,7 @@
+pyparsing<=1.5.7
 numpy
-rdflib
+git+https://github.com/RDFLib/rdflib@master
```

Comment:
Dependency from master is a recipe for disaster. The blame for this change https://github.com/vsoch/nidmviewer/blame/b93e91e12ddafc8056bae6d5a7917f20fcd21a4b/requirements.txt doesn't explain why existing stable version is not enough.